cli/utils/cert-resolver.js

```diff
@@ -0,0 +1,38 @@
+/*jslint node: true */
+'use strict';
+
+const fs = require('fs-extra');
+const os = require('os');
+const path = require('path');
+const logger = require('@blackbaud/skyux-logger');
+
+function getResolver() {
+  const certRootPath = path.resolve(`${os.homedir()}/.skyux/certs/`);
```

Comment:
This value is identical to the one in the CLI. Probably impossible, but do you think there's a way this value could live in one place? Maybe the path is passed into the `getResolver` function from the CLI?

Reply:
Great suggestion!